src/ppx/application.cpp

```diff
@@ -788,13 +788,18 @@ void Application::ShutdownMetrics()
 
 void Application::SaveMetricsReportToDisk()
 {
-    if (!mStandardOpts.pEnableMetrics->GetValue()) {
+    if (mStandardOpts.pEnableMetrics == nullptr || !mStandardOpts.pEnableMetrics->GetValue()) {
```

Comment:
Did you actually run into the case of this being nullptr? 
I don't see how that can happen, but I might be missing something. I think it should be asserted that it isn't nullptr instead.

Reply:
I've changed these to assertions. There's currently a crash at `HEAD` from a null pointer dereference in this function, and the assertions should improve the error messages if it happens again in the future.